kots-exporter/kots-exporter.sh

```diff
@@ -120,6 +120,17 @@ set_default_value(){
     then
         func="all"
     fi
+
+    # check if this is a dev migration
+    if [[ $dev == true ]]
+    then
+        export REGISTRY="devcharts"
+        echo "Using dev azure registry"
+    else
+        export REGISTRY="cciserver"
+    fi
+
+    export CHART="oci://devcharts.azurecr.io/circleci-server"
```

Comment:
This line should use the $REGISTRY value instead of hardcoding devcharts

Reply:
aah thanks. i missed that. good call out!